### PR TITLE
feat(AAPD-855): make sql appeal when mongo appeal is made

### PR DESCRIPTION
**Pull request by Si3rr4wow**
## Ticket Number

https://pins-ds.atlassian.net/browse/AAPD-

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/appeals-service-api/src/repositories/sql/appeals-repository.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	 * @returns {Promise<Appeal>}
+	 */
+	async createAppeal(appeal) {
+		try {
+			return await this.dbClient.appeal.create({
+				data: appeal
+			});
+		} catch (err) {
+			logger.error(err);
+			if (err instanceof Prisma.PrismaClientKnownRequestError) {
```
Comment (beejjacobs) [inline packages/appeals-service-api/src/repositories/sql/appeals-repository.js:31]: This check isn't enough to know that it's a duplicate (this was perhaps copied I missed it elsewhere).

See: https://www.prisma.io/docs/orm/reference/error-reference#prismaclientknownrequesterror - we can check error codes instead.

**Review by beejjacobs — COMMENTED** [review]
Small point on errors, but otherwise looks good to me.

File: packages/appeals-service-api/src/repositories/sql/appeals-repository.js (hunk 1 above)
Comment (Si3rr4wow) [reply packages/appeals-service-api/src/repositories/sql/appeals-repository.js:31]: interesting 🤔 I actually inherited this code by copy and pasting the user repo so I'll have a look at the link and fix it there too

**Review by beejjacobs — APPROVED** [review]
Looks good!

**Pull request merged**